lib/core/src/client/preview/start.tsx

```diff
@@ -0,0 +1,277 @@
+import { document, navigator, window } from 'global';
```

Comment:
How about we extract these files relating to the browser from lib/core?

Can they instead be given a place in client-api perhaps?

Reply:
I'm guessing, but all our preview-side React code (for docs) lives in `lib/core`. I don't know if there is a practical reason for that or not. 

Actually come to think of it, why do we even have React code running in the preview. Can we not do that (outside of docs at least), @shilman?

Reply:
What React code is there other than docs? A cursory search through the code doesn't yield much.

Reply:
As discussed:

It seems "bad" that we have a peer dep on `react[-dom]` and do a bunch of react rendering in `@storybook/core`. We should probably try and make the docs rendering (which does need React) only needed if you are using docs. However, that is out of scope of this refactor.

@ndelangen to your original point -- I think maybe the idea is that core uses these browser APIs and client-api *does not* (the original idea behind the split between core and client-api). The use case was React Native. I don't fully understand the original rationale in any case.

Also I think that's out of scope of this refactor (working out why) but this refactor would make it simple to move those files across.

Reply:
> Also I think that's out of scope of this refactor (working out why) but this refactor would make it simple to move those files across.

👍